### ISSUE TITLE
Documentation corresponding to Arguments for Pre-Trained Models inside <tf.keras.applications> is missing

### ISSUE THREAD

**Issue by rakeshmothukuru1** [tensorflow/tensorflow#37996]
## URL(s) with the issue: The information related to Arguments corresponding to the Pre-Trained Models defined under  https://www.tensorflow.org/api_docs/python/tf/keras/applications is missing.

Some examples links are shown below:

https://www.tensorflow.org/api_docs/python/tf/keras/applications/MobileNet

https://www.tensorflow.org/api_docs/python/tf/keras/applications/InceptionV3

https://www.tensorflow.org/api_docs/python/tf/keras/applications/ResNet50

https://www.tensorflow.org/api_docs/python/tf/keras/applications/VGG16

## Description of issue (what needs changing): The information corresponding to Arguments should be specified like that it is Specified in [Keras Website](https://keras.io/applications/#vgg16).

For example, why should someone use this method? How is it useful? : If someone want to know what Arguments should be passed while trying to use these Pre-Trained Models, information is lacking in TF.Org site and the Developers should go to Keras Website. The information is not available in the Source Code corresponding to those TF Pre-Trained Models.

### Correct links

Is the link to the source code correct? : Yes

### Parameters defined

Are all parameters defined and formatted correctly?

### Returns defined

Are return values defined? : Yes

### Usage example

Is there a usage example? : NO

### Submit a pull request? : No

Are you planning to also submit a pull request to fix the issue? See the docs
contributor guide: https://www.tensorflow.org/community/contribute/docs,
docs API guide: https://www.tensorflow.org/community/contribute/docs_ref and the
docs style guide: https://www.tensorflow.org/community/contribute/docs_style

**Comment by Byambaa0325** [comment]
Hello, I would like to work on this. Add docs and examples for the models, starting from the mobilenet.

**Comment by maxkaustav** [comment]
You can click on see nightly on the website to look at the arguments.

**Comment by rakeshmothukuru1** [comment]
@maxkaustav,
Thank you for your suggestion. But shouldn't the arguments visible in Stable Version as well, because, 

1. Many people refer the documentation of Stable Version (it will be open by default)
2. Documentation of Nightly Version will not be very neat, with Stricken Off lines, etc..

**Comment by Byambaa0325** [comment]
@rakeshmothukuru1,
Hi, I think the stricken off lines reflect the changes to the stable version, meaning it was deleted and will not be in the stable version when deployed. However, the documentation could still use some examples I believe

**Comment by ManishAradwad** [comment]
I will also begin with the documentation for VGG16. But what files should I edit?? Can someone plz help?

**Comment by Byambaa0325** [comment]
To whom it may concern, @gadagashwini (tagging because of the assignee label)

To prevent duplicated efforts, just notifying that I am working on adding examples and listing the raises for the models. I expect to finish within 48 hours for all the pretrained models. I'm adding to the docs in nightly. 

I'll also notify if I won't be able to complete it within the timeframe :)

**Comment by Byambaa0325** [comment]
I created a pull request for the examples I added at here [#38048](https://github.com/tensorflow/tensorflow/pull/38048)

**Comment by jvishnuvardhan** [comment]
@rakeshmothukuru1 I am closing this issue as the related PR https://github.com/tensorflow/tensorflow/pull/38048 was merged. Please feel free to reopen if the issue was not resolved. Thanks!

**Comment by rakeshmothukuru1** [comment]
@jvishnuvardhan,
You could have closed the issue with my consent.

I cannot Reopen the issue. Can you please reopen it because,

1. The PR has been raised and merged and some examples have been added in the PR but the same is not reflected in the [TF Site](https://www.tensorflow.org/api_docs/python/tf/keras/applications/VGG16) yet.

2. Information about the Arguments to be used for a particular Pre-Trained Model is still missing, corresponding to Stable Version.

**Comment by jvishnuvardhan** [comment]
@rakeshmothukuru1

**Comment by Byambaa0325** [comment]
The PR, that had basic examples, was merged. However, I can see that it was rolled back in the master by a member of TensorFlow team.
I guess they might already have been working on the docs but haven't put them out yet. I expect comprehensive documentation including the arguments for particular pre-trained models.

**Comment by lamberta** [comment]
FYI, nightly docs become the stable docs on next release

**Comment by mohantym** [comment]
Hi @rakeshmothukuru1! Concerned links in template has been updated with description of arguments in TF 2.6 , Please check the links from here. [MobileNet,](https://www.tensorflow.org/api_docs/python/tf/keras/applications/mobilenet/MobileNet)[InceptionV3](https://www.tensorflow.org/api_docs/python/tf/keras/applications/inception_v3/InceptionV3),[ResNet50](https://www.tensorflow.org/api_docs/python/tf/keras/applications/resnet50/ResNet50),[VGG16](https://www.tensorflow.org/api_docs/python/tf/keras/applications/vgg16/VGG16) .Closing this issue as it seems to be resolved. Thanks!
